src/LaunchDarkly.EventSource/EventSourceService.cs

```diff
@@ -154,7 +154,7 @@ private Encoding DetectEncoding(HttpResponseMessage response)
 
             while (!cancellationToken.IsCancellationRequested)
             {
-                string line = await DoTaskWithReadTimeout(() => reader.ReadLineAsync(), needManualTimeout);
+                string line = await DoTaskWithReadTimeout(reader.ReadLineAsync, needManualTimeout);
```

Comment:
This is just syntactic simplification - the method reference by itself does the same thing as a lambda that calls the method.